src/main/scala/org/apache/spark/streamdm/streams/FileReader.scala

```diff
@@ -97,35 +101,37 @@ class FileReader extends StreamReader with Logging {
   }
 
   /**
-   * Obtains the specification of the examples in the stream.
-   *
-   * @return an ExampleSpecification of the features
-   */
+    * Obtains the specification of the examples in the stream.
+    *
+    * @return an ExampleSpecification of the features
+    */
   override def getExampleSpecification(): ExampleSpecification = {
     init()
     spec
   }
 
   /**
-   * Get one Exmaple from file
-   *
-   * @return an Exmaple
-   */
+    * Get one Example from file
+    *
+    * @return an Example
+    */
   def getExampleFromFile(): Example = {
     var exp: Example = null
+    // start to read file from its beginning.
     if (lines == null || !lines.hasNext) {
+      //get the whole file
       lines = Source.fromFile(fileName).getLines()
     }
     // if reach the end of file, will go to the head again
     if (!lines.hasNext) {
-      lines = Source.fromFile(fileName).getLines()
+      println("=============================")

```

Comment:
Remove debug information.

Reply:
@nhnminh Can you please take a look on that? After that, I will provide my review. Thanks!

Reply:
Hi, I changed the debug information as suggested. I will have some changes in this file for the next commit.